src/eigen/lobpcg_hyper_impl.jl

```diff
@@ -149,7 +181,7 @@ end
 # (which implies that X'BX is relatively well-conditioned, and
 # therefore that it is safe to cholesky it and reuse the B apply)
 function B_ortho!(X, BX)
-    O = Hermitian(X'*BX)
+    O = Hermitian(mul_hermi(X'*BX))
```

Comment:
Oh also ... if we put the `Hermitian` wrapper inside the `mul_hermi` function, than we can even avoid computing the lower triangle at all as the wrapper will make sure it's never accessed.